tests/etc/commands.yml.example

```diff
@@ -282,22 +282,6 @@ keymile_nos:
   save_config_confirm: True
   save_config_response: '[OK]'
 
-dlink_ds:
-  version: "show greeting_message"
-  basic: "show ipif"
-  config:
-    - enable command logging
-  config_verification: "show config current_config include \"logging\""
-  extended_output: "show config current_config"   # requires paging to be disabled
-
-dlink_ds_telnet:
-  version: "show greeting_message"
-  basic: "show ipif"
-  config:
-    - enable command logging
-  config_verification: "show config current_config include \"logging\""
-  extended_output: "show config current_config"   # requires paging to be disabled
-
```

Comment:
Rebase to fix this accidental removal.

Reply:
Please check my new commit, should of fixed it